designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -255,10 +259,11 @@ fun SquooshRoot(
             else -> ""
         }
 
-    val layoutIdAllocator = remember { SquooshLayoutIdAllocator() }
-    val rootLayoutId = remember(docId) { SquooshLayout.getNextLayoutId() * 100000000 }
+    val layoutManager = remember(docId) { SquooshLayout.newLayoutManager() }
+    val layoutIdAllocator = remember(docId) { SquooshLayoutIdAllocator() }
     val layoutCache = remember(docId) { HashMap<Int, Int>() }
     val layoutValueCache = remember(docId) { HashMap<Int, Layout>() }
+    val rootLayoutId = 0 // XXX: Remove -- we needed this when there was only one layout manager.
```

Comment:
remove?

Reply:
Yep, removed, and also cleaned up the rest of the references to `rootLayoutId` since it doesn't really exist now.